change-notes/support/language-support.rst

```diff
@@ -13,7 +13,9 @@ Note that where there are several versions or dialects of a language, the suppor
 
 .. container:: footnote-group
 
-    .. [1] The best results are achieved with COBOL code that stays close to the ANSI 85 standard.  
-    .. [2] Builds that execute on Java 6 to 11 can be analyzed. The analysis understands Java 11 language features.
-    .. [3] JSX and Flow code, YAML, JSON, and HTML files may also be analyzed with JavaScript files. 
-    .. [4] TypeScript analysis is performed by running the JavaScript extractor with TypeScript enabled. This is the default for LGTM.   
+    .. [1] Support for the Arm Compiler (armcc) is preliminary.
+    .. [2] In addition, support for the preview features of C#8 and .NET Core 3.0.
```

Comment:
space between C# and 8 ?

Reply:
Sorry, it doesn't follow the C scheme of C89 C++11 etc.

Reply:
Thanks. I'll update it to add the space shortly.

Reply:
Should it be "C# 8.0"  or is "C# 8" okay?